hooks/publish_file.py

```diff
@@ -598,6 +599,11 @@ def _get_publish_path(self, settings, item):
                     "Used publish template to determine the publish path: %s" %
                     (publish_path,)
                 )
+        elif not work_template and publish_template:
```

Comment:
The conditional is confusing. I think just do an `else:` and debug log the value of `publish_template` and `work_template`.  We def shouldn't warn here since templates aren't required for zero config workflows which also use this plugin.

Reply:
Ah, I hadn't thought about that. Good point!